libafl_derive/src/lib.rs

```diff
@@ -69,3 +69,89 @@ pub fn libafl_serdeany_derive(input: TokenStream) -> TokenStream {
         libafl_bolts::impl_serdeany!(#name);
     })
 }
+
```

Comment:
i see there are the changes from #2339.
not a problem, but ideally we should rebase on this once it's merged.